nutrition_labels/grant_tagger.py

```diff
@@ -76,18 +87,34 @@ def evaluate(self, X, y):
         print(classification_report(y, y_predict))
         print(confusion_matrix(y, y_predict))
 
+    def return_mislabeled_data(self,y_test,y_pred):
+        X_test = [self.X[i] for i in self.test_indices]
+        X_test_df = pd.DataFrame({'Description': X_test,
+                                'True_label':y_test,
+                                'Predicted_label':y_pred})
+        mislabeled = X_test_df[X_test_df['True_label'] != X_test_df['Predicted_label']]
+        return(mislabeled)
+
+
+    def inverse_transform(self,X):
+        return self.vectorizer.inverse_transform(X)
+
+
 
 if __name__ == '__main__':
     
     data = pd.read_csv('data/processed/training_data.csv')
 
-    grant_tagger = GrantTagger(sample_4s=50, ngram_range=(1,2), test_size=0.25)
+    grant_tagger = GrantTagger(sample_4s=50, ngram_range=(1,2), test_size=0.25, random_state= 4)
     X_train, X_test, y_train, y_test = grant_tagger.transform(data)
     grant_tagger.fit(X_train, y_train)
 
     print("Evaluate training data")
     grant_tagger.evaluate(X_train, y_train)
     print("Evaluate test data")
     grant_tagger.evaluate(X_test, y_test)
+    print("Mislabled Grant descriptions")
+    print(grant_tagger.return_mislabeled_data(y_test, grant_tagger.predict(X_test)))
```

Comment:
if you make the third argument addition:
```
print(grant_tagger.return_mislabeled_data(y_test, grant_tagger.predict(X_test), grant_tagger.test_indices))
```
and that way you can also investigate the results of
```
print(grant_tagger.return_mislabeled_data(y_train, grant_tagger.predict(X_train), grant_tagger.train_indices))
```
which might give you some sort of insight into the training